milou_ngi_config.yaml

```diff
@@ -52,3 +73,20 @@ slurm:
 
 supported_genomes:
     "GRCh37": /proj/a2014205/piper_references/gatk_bundle/2.8/b37/human_g1k_v37.fasta
+
+test_data:
+    workflows:
+        whole_genome_reseq:
+            test_project:
+                project_id: P0000
+                project_name: "Y.Mom_15_01"
+                bpa: "whole_genome_reseq"
+            customize_config:
+                analysis:
+                    top_dir: /proj/a2014205/nobackup/NGI/test_data/
+                charon:
+                    charon_base_url: http://charon-dev.scilifelab.se
```

Comment:
Hm. This won't work when we're trying to write back to charon-dev later. Ima think on this.